Agreements/Airport/BB_TCCOW.txt

```diff
@@ -1,4 +1,6 @@
+COPX:EGBB:*:*:ADMEX:*:BBAPP:London TC COWLY:6000:*:ADMEX
 COPX:EGBB:*:*:COWLY:*:BBAPP:London TC COWLY:6000:*:COWLY
 COPX:EGBB:*:*:DTY:*:BBAPP:London TC COWLY:6000:*:DTY
 COPX:EGBB:*:*:CPT:*:BBAPP:London TC COWLY:6000:*:CPT
-COPX:EGBB:*:*:WCO:*:BBAPP:London TC COWLY:6000:*:WCO
\ No newline at end of file
+COPX:EGBB:*:*:WCO:*:BBAPP:London TC COWLY:6000:*:WCO
+COPX:EGBB:*:*:UNGAP:*:BBAPP:London TC COWLY:6000:*:UNGAP
```

Comment:
Do these definitely work if they are not restricted to a climb of 6000ft using the FIX - rather than FIXAFTER - parameter? I can imagine on runway 33 that EuroScope might predict they will climb into the Stafa sector before entering TC COWLY?

Reply:
Does someone have a possibility to test these today/tomorrow?

Reply:
These work fine